pkg/report/guilty.go

```diff
@@ -52,3 +55,23 @@ nextFile:
 	}
 	return ""
 }
+
+func GetMaintainers(linux string, file string) ([]string, error) {
+	// get_maintainer.pl is a Linux kernel script.
+	cmd := exec.Command("./scripts/get_maintainer.pl", "-f", file)
+	cmd.Dir = linux
+	stderr := new(bytes.Buffer)
```

Comment:
Do we really need stdout/stderr separately? I think we don't because we need to be prepared to parse any garbage in output anyway.
Then, there is `cmd.CombinedOutput` function that runs the command and returns output.
Or, better, `osutil.RunCmd` that can also accepts dir and timeout.